src/reducers/tests/actions/resetActions.test.jsx

```diff
@@ -0,0 +1,65 @@
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import moxios from 'moxios';
+import resetPasswordActions from
+  '../../../actions/resetActions/resetPasswordActions';
+import * as types from '../../../actions/types';
+import Response, { errorResponse } from './moxios_mock';
+
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+jest.mock('react-notify-toast');
+
+describe('Reset password', () => {
+  describe('actions', () => {
+    const email = 'dorothy.asiimwe@andela.com';
```

Comment:
Use a dummy email addresses here instead of your real email addresses

Reply:
noted